cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -251,7 +251,7 @@ def continue_cherry_pick(self):
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.version_option(version="0.0.2.post2")
+@click.version_option(version="0.1.0.dev1")
```

Comment:
Hmm.. it's an old artifact :stuck_out_tongue_closed_eyes: 
Thanks :) I'll remove `__init__.py` and move `__version__` to `cherry_picker.py`